cli/js/os.ts

```diff
@@ -166,6 +167,13 @@ type DirKind =
  * | macOS   | `$HOME`/Library/Preferences           | /Users/Alice/Library/Preferences |
  * | Windows | `{FOLDERID_RoamingAppData}`           | C:\Users\Alice\AppData\Roaming   |
  *
+ * "executable"
+ * |Platform | Value                                                           | Example                |
+ * | ------- | --------------------------------------------------------------- | -----------------------|
+ * | Linux   | `XDG_BIN_HOME` or `$XDG_DATA_HOME`/../bin or `$HOME`/.local/bin | /home/alice/.local/bin |
+ * | macOS   | -                                                               | -                      |
+ * | Windows | -                                                               | -                      |
```

Comment:
This only works on linux?

Reply:
Yes but I don't think that should stop us from exposing it here We're just extending these from `dirs`, right? Requiring at least two platforms seems arbitrary and it may exist for other platforms in the future.